SU-SWS/stanford_publication
# D8CORE-3948: fix to the more publication cards and added the see all …

…publications link

# READY FOR REVIEW

# Summary
- Align the cards on the publications node
- Add the See All Publications button

# Needed By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. Pull in the configuration change: https://github.com/SU-SWS/stanford_profile/pull/385
3. Import the config and clear all the caches.
4. Verify the More Publications cards on a node are all top aligned.
5. Verify there is a button with a link to /publications

# Affected Projects or Products
- Publications

# Associated Issues and/or People
- D8CORE-3948
- https://github.com/SU-SWS/stanford_profile/pull/385

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
